homeassistant/components/automation/__init__.py

```diff
@@ -148,36 +153,16 @@ def trigger(hass, entity_id=None):
     hass.services.call(DOMAIN, SERVICE_TRIGGER, data)
 
 
+def reload(hass):
```

Comment:
It's part of the `automation` component. So to use it you would:

```python
from homeassistant.components import automation
automation.reload(hass)
```

Which is in line with the `turn_on`, `turn_off` and `toggle` methods under switch, light, etc